### PR TITLE
[nodejs] switch nodejs weblog images to alpine

### PR DESCRIPTION
**Pull request by rochdev**
## Motivation

Alpine is a lot smaller which will make pulling the images much faster.

## Changes

Switch Node.js weblog images to alpine.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Review by Qard — APPROVED** [review]

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
There are some test failures related with this PR

File: utils/build/docker/nodejs_otel/express4-otel.Dockerfile [hunk 1]
```diff
@@ -32,10 +34,11 @@ RUN npm install @opentelemetry/instrumentation-mysql2
...
-RUN echo "1.0.0" > SYSTEM_TESTS_LIBDDWAF_VERSION
-RUN echo "1.0.0" > SYSTEM_TESTS_APPSEC_EVENT_RULES_VERSION
+RUN printf "1.0.0" > SYSTEM_TESTS_LIBDDWAF_VERSION
+RUN printf "1.0.0" > SYSTEM_TESTS_APPSEC_EVENT_RULES_VERSION
 
 # docker startup
-RUN echo '#!/bin/bash\nnode --require @opentelemetry/auto-instrumentations-node/register app.js' > app.sh
+RUN printf '#!/bin/bash\nnode --require @opentelemetry/auto-instrumentations-node/register app.js' > app.sh
 RUN chmod +x app.sh
+RUN ls -lisa /usr/app/node_modules
```
Comment (simon-id) [inline utils/build/docker/nodejs_otel/express4-otel.Dockerfile:43]: new line for debug or forgot to remove ?

**Comment by rochdev** [comment]
> Yeah that looks like an unrelated problem actually, no idea why standalone appsec tests would fail only on alpine

@simon-id I rebased and it's still failing, but it's not failing on `main`. For some reason locally for me it fails on every branch, but in CI it's only in this branch which seems to indicate some problem between AppSec and Alpine.

**Review by simon-id — APPROVED** [review]

**Comment by rochdev** [comment]
@robertomonteromiguel Everything is passing now.

**Review by robertomonteromiguel — APPROVED** [review]

**Pull request merged**